feature/gribi/otg_tests/ack_in_presence_other_routes/route_ack_test.go

```diff
@@ -286,7 +281,7 @@ func TestRouteAck(t *testing.T) {
 	configureNetworkInstance(t)
 	t.Logf("Configure the DUT with static route 203.0.113.0/24...")
 	dutConf := configStaticRoute(t, dut, ateDstNetCIDR, staticNH)
-	dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance).Protocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "STATIC").Static(ateDstNetCIDR).Replace(t, dutConf)
+	dut.Config().NetworkInstance(*deviations.DefaultNetworkInstance).Protocol(telemetry.PolicyTypes_INSTALL_PROTOCOL_TYPE_STATIC, "DEFAULT").Update(t, dutConf)
```

Comment:
Would it make sense to have this update in the configStaticRoute function itself?

Reply:
Agreed, moved it under configStaticRoute